conf/messages.en

```diff
@@ -6,6 +6,10 @@ site.continue = Continue
 site.back = Back
 site.save_and_continue = Save and continue
 site.change = Change
+site.change.hint.disassociate.ucr = Change DUCR dissociation
+site.change.hint.associate.ucr = Change DUCR association
+site.change.hint.associate.mucr = Change MUCR association
+site.change.hint.shut.mucr = Change MUCR
```

Comment:
I would rather use `disassociateUcr` etc. Using dot as a space does not fit in here.

Reply:
I think `disassociate.ucr` reads better than `disassociateUcr`. I have also used the same ID for the `movements` service.